.github/workflows/admin-ci.yml

```diff
@@ -11,6 +11,7 @@ env:
   APP_KEY: NOT_AN_REAL_APP_KEY
   ARCWELL_INSTANCE_ID: arcweb-ci
   ARCWELL_INSTANCE_NAME: CI Arcwell Server"
+  ARCWELL_REGISTER_ENABLED: true
```

Comment:
@mberlin-arcweb I updated this list to be alphabetical order last PR.  Please move this to proper order

Reply:
fixed